allauth/account/utils.py

```diff
@@ -362,16 +363,25 @@ def passthrough_next_redirect_url(request, url, redirect_field_name):
 
 
 def user_pk_to_url_str(user):
+    """
+    This should return a string.
+    """
+    User = get_user_model()
+    if hasattr(models, 'UUIDField') and issubclass(type(User._meta.pk), models.UUIDField):
+        return user.pk.hex
+
     ret = user.pk
     if isinstance(ret, six.integer_types):
         ret = int_to_base36(user.pk)
-    return ret
+    return str(ret)
```

Comment:
yes.  from a squashed merge before i realized that I had to return hex.  will fix it.